src/Raven.Client/Documents/Session/Loaders/IncludeBuilder.cs

```diff
@@ -331,5 +375,50 @@ private void WithAlias(Expression<Func<T, string>> path)
             if (Alias == null)
                 Alias = path.Parameters[0].Name;
         }
+
+        private void IncludeTimeSeries(string path, string name, DateTime from, DateTime to)
```

Comment:
what is the meaning of the path here?

Reply:
we can include time series of related documents by path